src/constants/Superteam.ts

```diff
@@ -1,6 +1,16 @@
 import { Regions } from '@prisma/client';
 
 export const Superteams = [
+  {
+    name: 'Global',
+    icons: '',
+    banner: '',
+    region: Regions.GLOBAL,
+    displayValue: 'Global',
+    country: ['Global'],
+    code: 'GLB', //use this in country.ts to get global icon once added in their github.
```

Comment:
no need; follow the approach we used for Balkans

Reply:
talking about the country.ts comment

Reply:
So should I just remove GLB  and do "GLOBAL"?